app/models/manageiq/providers/ibm_power_hmc/inventory/collector/infra_manager.rb

```diff
@@ -52,8 +53,19 @@ def vnics
     @vnics || {}
   end
 
+  def ssps
+    @ssps || {}
+  end
+
   private
 
+  def do_ssps(connection)
+    @ssps = {}
+    @ssps = connection.ssps
+    rescue IbmPowerHmc::Connection::HttpError => e
```

Comment:
This rescue should be inline with the method declaration if you're not explicitly declaring the begin